src/index.js

```diff
@@ -132,6 +133,9 @@ export default class StepWizard extends Component {
         window.location.hash = this.state.hashKeys[activeStep];
     }
 
+    // Allows for using HTML elements as a step
+    isReactComponent = type => typeof type === 'function'
```

Comment:
Nothing against your fix, just wanted to point out that `Component.prototype.isReactComponent` is provided by React itself. Checkout [this article](https://overreacted.io/how-does-react-tell-a-class-from-a-function/) and [this relevant code](https://github.com/facebook/react/blob/769b1f270e1251d9dbdce0fcbd9e92e502d059b8/packages/react-reconciler/src/ReactFiber.js#L297-L300) from React source.

Reply:
Oh nice! That seems like it would be a more stable option.